src/ethereum_test_tools/common/types.py

```diff
@@ -1028,6 +1027,25 @@ def default(self, obj):
                 ],
             )
         elif isinstance(obj, FixtureBlock):
+            b_txs = (
```

Comment:
I think this part can be completely removed by using something like:
```
to_json_or_none(obj.txs)
```
Which in theory should simply reuse the code of the `elif isinstance(obj, Transaction):` branch in this same function.

The `even_padding` could also be added in that part of the code.

Reply:
That was my original plan but the fields expected by `t8n.evaluate` within `blockchain_test.py/make_block()`, via `block.txs`:

![image](https://user-images.githubusercontent.com/60348173/216666739-7b48285b-ecc9-4d9c-a8ab-8535c1c215a4.png)

are different to those required for the fixture format. For example `t8n` wants `"gas"` but the fixture format requires `"gasPrice"`, hence it was not possible to use the exact same `Transaction` type. As a workaround, the `Transaction` type is pre-processed as `b_txs` during the json formatting of a `FixtureBlock` hence fixture format of blocks are correct.

![image](https://user-images.githubusercontent.com/60348173/216667498-d1b83051-b597-4e1f-891f-5c0d39b5c942.png)

It would be useful if the format was standardised